iwyu_globals.cc

```diff
@@ -228,6 +231,7 @@ int CommandlineFlags::ParseArgv(int argc, char** argv) {
     {"error_always", optional_argument, nullptr, 'a'},
     {"debug", required_argument, nullptr, 'd'},
     {"regex", required_argument, nullptr, 'r'},
+    {"canonical_stdlib", no_argument, nullptr, 's'},
```

Comment:
We should be careful with flags, because they need to stick around for backward compatibility.

I wonder if there should be something like `--experimental=<feature-name>` where we promise zero backward compat. Like `--debug`. Probably not worth the effort here, but interesting to consider.